app/routes.py

```diff
@@ -1,2 +1,77 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request, Response
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["GET", "POST"])
+def handle_planets():
+    if request.method == "GET":
+
+        name_query = request.args.get("name")
+        if name_query:
+            planets = Planet.query.filter_by(name=name_query)           
+        else:
+            planets = Planet.query.all()
+
+        planets_response = []
+        for planet in planets:
+            planets_response.append({
+                "id": planet.id,
+                "name": planet.name,
+                "description": planet.description
+            })
+        return jsonify(planets_response)
+    
+    elif request.method == "POST":
+        request_body = request.get_json()
+        new_planet = Planet(name=request_body["name"],
+                            description=request_body["description"])
```

Comment:
Would be good to add in some error handling here since request_body is the data that a client sends via the request body. We should assume that requests could have bad data, either missing keys or values or the keys could be misspelled.

If the request_body didn't have "name" but had "planet_name" then line 28 would throw an unhandled exception.